client/src/views/pages/home.js

```diff
@@ -10,7 +10,9 @@ import background1 from '../../assets/img/backgrounds/background-1.jpg';
 import background2 from '../../assets/img/backgrounds/background-2.jpg';
 import background3 from '../../assets/img/backgrounds/background-3.jpg';
 
-import placeholder from '../../assets/img/misc/placeholder-1.png';
+import placeholder1 from '../../assets/img/misc/home_image1.png';
+import placeholder2 from '../../assets/img/misc/placeholder-1.png';
+import placeholder3 from '../../assets/img/misc/placeholder-2.png';
```

Comment:
Should rename these if they're not placeholders anymore.

Should also rename the first file to `home-1.png`. There's also a background image in this PR with an `_`, but I cannot see where that's imported. Can you rename it please and where is it actually being used?

Reply:
- [x] Renamed
- [x] first img home-1.png
- [x] __1 was the old bgimg for the home.js it is deleted now.